### PR TITLE
Fix Lint/UnusedBlockArgument offenses

### PR DESCRIPTION
**Pull request by tagliala**
### Description
Please describe your pull request. Thank you for contributing! You're the best.

### Your checklist for this pull request

- [x] I have reviewed the [guidelines for contributing](../blob/master/CONTRIBUTING.md) to this repository.
- [ ] I have added (or updated) appropriate tests if this PR fixes a bug or adds a feature.
- [ ] If this PR doesn't need tests (docs change), I added `[ci skip]` to the title of the PR.
- [ ] If this closes any issues, I have added "Closes `#issue`" to the PR description or my commit messages.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [ ] I added an entry to the [changelog](../blob/master/CHANGELOG.md).

### CODE REVIEW

File: lib/axlsx/workbook/worksheet/pivot_table.rb [hunk 1]
```diff
@@ -220,7 +220,7 @@ def to_xml_string(str = +'')
           str << '<colFields count="1"><field x="-2"/></colFields>'
           str << "<colItems count=\"#{data.size}\">"
           str << '<i><x/></i>'
-          data[1..-1].each_with_index do |datum_value, i|
+          (data.size - 1).times do |i|
```
Comment (tagliala) [inline lib/axlsx/workbook/worksheet/pivot_table.rb:223]: please check this

File: lib/axlsx/workbook/worksheet/pivot_table.rb [hunk 2]
```diff
@@ -220,7 +220,7 @@ def to_xml_string(str = +'')
...
             str << "<i i=\"#{i + 1}\"><x v=\"#{i + 1}\"/></i>"
           end
```
Comment (tagliala) [inline lib/axlsx/workbook/worksheet/pivot_table.rb:225]: Please check this method, maybe something is wrong here and `v` should contain something else?

Comment (tagliala) [reply lib/axlsx/workbook/worksheet/pivot_table.rb:225]: reverting to draft to prevent unwanted merges

**Review by kiskoza — APPROVED** [review]

File: lib/axlsx/workbook/worksheet/pivot_table.rb [hunk 3]
```diff
@@ -220,7 +220,7 @@ def to_xml_string(str = +'')
           str << '<colFields count="1"><field x="-2"/></colFields>'
           str << "<colItems count=\"#{data.size}\">"
           str << '<i><x/></i>'
-          data[1..-1].each_with_index do |datum_value, i|
+          (data.size - 1).times do |i|
             str << "<i i=\"#{i + 1}\"><x v=\"#{i + 1}\"/></i>"
```
Comment (tagliala) [inline lib/axlsx/workbook/worksheet/pivot_table.rb:224]: @kiskoza so this is ok?

Comment (kiskoza) [reply lib/axlsx/workbook/worksheet/pivot_table.rb:224]: yes, `v` does not hold the data here, it's used later between line 244-254

**Pull request merged**